### PR TITLE
refactor(translation): correct german translation

### PR DESCRIPTION
**Pull request by 0x04**
This inludes:

* Change "lügende Person" zu "Lügner"
* Change "ermittelende Person" zu "Ermittler"
* Correct upper and lower case
* Correct punctation marks
* Correct content errors

Related to #2  and #7

### CODE REVIEW

**Comment by Haroenv** [comment]
we didn't use Lünger & Ermittler because they're male, and otherwise it would need to be something like Lügner.in which doesn't read nice to me

**Comment by 0x04** [comment]
Okay, didn't thought on that. "Lügner" & "Ermittler" was sounding more natural for me.

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "intro": {
     "text": [
-      "In diesem Spiel gibt es zwei Rollen: Den Lügner und den Ermittler. Bei Beginn jeder Runde werden die Rollen getauscht.",
-      "Dem Lügner werden zwei zufällige Wikipedia Artikel präsentiert. Nur einer der beiden Artikel kann gelesen werden.",
-      "Der Ermittler sieht die Titel beider Artikel und bekommt die Möglichkeit, Fragen zu den Artikeln zu stellen.",
-      "Wird der Ermittler herausfinden, welchen Artikel sich der Lügner ausgedacht hat?"
+      "In diesem Spiel gibt es zwei Rollen: Die lügende Person und die ermittelnde Person. Bei Beginn jeder Runde werden die Rollen getauscht.",
```
Comment (0x04) [inline public/locales/de/translation.json:4]: I changed the order of liar/investigator to reflect which person plays which role first. Perhaps that should taken over for the other translations, too?

**Comment by dervondenbergen** [comment]
I'm not that good with punctation 😅

**Pull request merged**